frontend/src/pages/NewRunV2.tsx

```diff
@@ -246,31 +311,58 @@ function NewRunV2(props: NewRunV2Props) {
       <div className={commonCss.scrollContainer}>
         <div className={commonCss.header}>Run details</div>
 
+        {!pipelineId && apiResourceRefFromRun && !apiResourceRefFromRun[1] && (
+          <div>
+            <div>
+              <span>{usePipelineFromRunLabel}</span>
+            </div>
+            <div className={classes(padding(10, 't'))}>
+              {originalRunId && (
+                <Link className={classes(commonCss.link)} to={pipelineDetailsUrl}>
```

Comment:
Enabling displaying the pipeline details for SDK-created run will be in the later PR.